static/app/views/monitors/components/overviewTimeline/index.tsx

```diff
@@ -132,17 +163,26 @@ function MonitorDetails({monitor}: {monitor: Monitor}) {
 
 const MonitorListPanel = styled(Panel)`
   display: grid;
-  grid-template-columns: 350px 1fr;
+  grid-template-columns: 350px 100px 1fr;
 
   a,
-  a + div {
+  a + div,
+  a + div + div {
     transition: background 50ms ease-in-out;
   }
 
+  /* Highlights row when hovering <a> */
   a:hover,
   a:hover + div,
+  a:hover + div + div,
+  /* Highlights row when hovering env <div> */
   a:has(+ div:hover),
-  a + div:hover {
+  a + div:hover,
+  a + div:hover + div,
+  /* Highlights row when hovering timeline <div> */
```

Comment:
woof haha

Reply:
yea 😓 it's getting crazy here...

Reply:
also I realized, this hover highlight doesn't actually work in firefox. `has` is not supported.

Reply:
dang should we just have some kind of overlay element that exists across all rows that is visible on hover?

Reply:
Yeah we'll probably have to do something like that 😢

Reply:
:'( rip, i can work on that in a follow up

Reply:
https://github.com/getsentry/sentry/pull/53117/files